scaling_performance/host_practices.adoc

```diff
@@ -0,0 +1,106 @@
+[[scaling-performance-capacity-host-practices]]
+= Recommended Host Practices
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+
+[[scaling-performance-capacity-host-practices-master]]
+== Recommended Practices for {product-title} Master Hosts
+
+Besides pod traffic, the most-used data-path in an {product-title}
+infrastructure is between the {product-title} master hosts and etcd. The
+{product-title} API server (part of the master binary) consults etcd for node
+status, network configuration, secrets, and much more.
+
+Optimize this traffic path by:
+
+* Co-locating master hosts and etcd servers.
+
+* Ensuring an uncongested, low latency LAN communication link between master hosts.
+
+* Ensuring the first etcd server listed in
+ *_/etc/origin/master/master-config.yaml_* is the local etcd instance.
+
+[[scaling-performance-capacity-host-practices-node]]
+== Recommended Practices for {product-title} Node Hosts
+
+The {product-title} node configuration file at
+*_/etc/origin/node/node-config.yaml_* contains important options such as the
+iptables synchronization period, the Maximum Transmission Unit (MTU) of the SDN network, and the proxy-mode.
+
+The node configuration file allows you to pass arguments to the kubelet
+(node) process. You can view a list of possible options by running `kubelet
+--help`.
+
+[NOTE]
+====
+Not all kubelet options are supported by {product-title}.
+====
```

Comment:
Hmm this is leftover from the Whitepaper. @jeremyeder any chance you can shed some light on this? I'm guess it should be something like:

"You can view a list of possible options by running `kubelet --help`, but note that as it's the upstream command, not all options are supported by OpenShift. See X for a list of supported options."

But I'm not sure?